### PR TITLE
[MIG] stock_orderpoint_generator to 14.0

### PR DESCRIPTION
**Pull request by sergiocorato**

### CODE REVIEW

File: stock_orderpoint_generator/models/orderpoint_template.py [hunk 1]
```diff
@@ -184,8 +184,9 @@ def create_orderpoints(self, products):
         """Create orderpoint for *products* based on these templates.
         :type products: recordset of products
         """
-        self._disable_old_instances(products)
```
Comment (rousseldenis) [inline stock_orderpoint_generator/models/orderpoint_template.py]: @sergiocorato Why this change ? Is there a problem using the recordset ?

Comment (sergiocorato) [reply stock_orderpoint_generator/models/orderpoint_template.py]: I fixed it, thanks.

**Comment by sergiocorato** [comment]
I added missing security rule for TransientModel.

**Review by elvise — APPROVED** [review]
Functional LGTM!

**Comment by francesco-ooops** [comment]
@rousseldenis good to go?

**Review by milinsoft — APPROVED** [review]

**Review by ilyasProgrammer — APPROVED** [review]
LGTM

**Comment by francesco-ooops** [comment]
@OCA/logistics-maintainers could this be merged? thanks!

File: stock_orderpoint_generator/models/orderpoint_template.py [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+class OrderpointTemplate(models.Model):
+    """Template for orderpoints
+
+    Here we use same model as stock.warehouse.orderpoint but set product_id
+    as non mandatory as we cannot remove it. This field will be ignored.
+
+    This has the advantage of ensuring that the order point
+    and the order point template have the same fields.
+
+    _table is redefined to separate templates from orderpoints
```
Comment (simahawk) [inline stock_orderpoint_generator/models/orderpoint_template.py:21]: this is not needed as the model is definitely different
